SystemServices/SystemServices.cpp

```diff
@@ -3874,12 +3877,17 @@ namespace WPEFramework {
             int seconds = 600; /* 10 Minutes to Reboot */
 
             LOGINFO("len = %lud\n", len);
+            int state = 0;
+	    	IARM_Bus_SYSMgr_SystemState_t stateId;
+	    	IARM_Bus_SYSMgr_EventData_t *sysEventData = nullptr;
             /* Only handle state events */
             if (eventId != IARM_BUS_SYSMGR_EVENT_SYSTEMSTATE) return;
 
-            IARM_Bus_SYSMgr_EventData_t *sysEventData = (IARM_Bus_SYSMgr_EventData_t*)data;
-            IARM_Bus_SYSMgr_SystemState_t stateId = sysEventData->data.systemStates.stateId;
-            int state = sysEventData->data.systemStates.state;
+            if(data != nullptr){
+                sysEventData = (IARM_Bus_SYSMgr_EventData_t*)data;
+                stateId = sysEventData->data.systemStates.stateId;
```

Comment:
please remove your changes. stateId is now uninitialized. you should avoid changing code for tests, especially like this

Reply:
Now stateId is initialized, this check is required what if data is nullptr?